1. CWE Base Mod/Cold War Era Mod (CWE)/common/production_methods/bg_government_1_goods.txt

```diff
@@ -320,6 +322,9 @@ pm_espionage_inactive = {
 	
 }
 
+# This should provide some other output
+# or it's effectively useless
```

Comment:
This is meant to represent the cost of having a secret service, other than just requiring more bureaucrats